.github/workflows/publish-python-wrapper-client.yaml

```diff
@@ -32,4 +33,34 @@ jobs:
         packages-dir: clients/python-wrapper/dist/
         verbose: true
 
-# TODO: Publish docs once we decide on documentation process
+    - name: Extract docs version
+      working-directory: ./clients/python-wrapper/
+      shell: bash
+      run: |
+          echo "tag=$(python setup.py --version)" >> $GITHUB_OUTPUT
+      id: docver
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
```

Comment:
- use string for python version - just so yaml will not parse it as float
- cache using pip
-  
```
steps:
- uses: actions/checkout@v4
- uses: actions/setup-python@v4
  with:
    python-version: '3.9'
    cache: 'pip'
    cache-dependency-path: 'clients/python-wrapper/requirements.txt'
- run: pip install -r requirements.txt
```